src/main/java/teammates/ui/controller/AdminEmailComposePageData.java

```diff
@@ -17,6 +17,14 @@ public AdminEmailComposePageData(AccountAttributes account) {
         state = AdminEmailPageState.COMPOSE;
     }
 
+    public AdminEmailComposePageData(AccountAttributes account, FileUploadPageData data) {
+        super(account);
+        this.isFileUploaded = data.isFileUploaded;
+        this.fileSrcUrl = data.fileSrcUrl;
+        this.ajaxStatus = data.ajaxStatus;
```

Comment:
You may consider putting these three lines on the call site instead instead of creating a dedicated constructor just for one time usage.